config/jobs/kubernetes/community/community-presubmit.yaml

```diff
@@ -1,22 +1,20 @@
 presubmits:
   kubernetes/community:
   - name: pull-community-verify
+    decorate: true
     branches:
-    - master
+    - ^master$
     always_run: true
     labels:
       preset-service-account: "true"
     spec:
       containers:
       - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+        command:
+        - "/bin/bash"
```

Comment:
Do we need to route this through bash?

Reply:
this is why I'm asking for review on the PR... since this is using podutils, we need to give this an explicit command

I didn't want to invoke the entrypoint of `/runner` since that clones test-infra and I can't see that we'd want that.  Most prior art I've seen by looking through jobs thus far involves running `bash -c`

Reply:
I know we invoke make directly on the typecheck job that runs in the kubekins image: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/typecheck.yaml

Reply:
My understanding is that `bash -c` is only needed if the command and/or args need to be evaluated by the shell. e.g. chaining commands with `&&` or `|`, expanding env vars not defined in the container spec env, or redirecting input/output streams. If programs expect to be run from a shell that might also make a difference, but I'm not sure if thats a thing?
In general if you have a plain binary name and arguments that don't need to be evaluated it shouldn't be needed so I think the following would work:
```yaml
command:
- make
args:
- verify